doc/user_manual/postprocessor.tex

```diff
@@ -1702,10 +1679,12 @@ \subsubsection{ETImporter}
 
 \subsubsection{Metric}
 \label{MetricPP}
-The \textbf{Metric} post-processor is specifically used to calculate the distance values among points and histories,
+The \textbf{Metric} post-processor is specifically used to calculate the distance values among points from PointSet and histories from HistorySet,
```

Comment:
PointSet and HistorySet should be plural, if I understand your meaning.  If not, then it should be "among points from a PointSet" and etc.

Reply:
fixed.